packages/angular/cli/src/command-builder/schematics-command-module.ts

```diff
@@ -150,6 +152,32 @@ export abstract class SchematicsCommandModule
         ]);
       }
 
+      // Handle `"format": "path"` options.
+      const schema = schematic?.schemaJson;
+      if (!options || workingDir === '' || !schema || !isJsonObject(schema)) {
+        return options;
+      }
+
+      for (const [key, value] of Object.entries(options)) {
+        if (value !== undefined) {
+          continue;
+        }
+
+        const properties = schema?.['properties'];
+        if (!properties || !isJsonObject(properties)) {
+          break;
+        }
+
+        const property = properties[key];
+        if (!property || !isJsonObject(property)) {
+          continue;
+        }
+
+        if (property['format'] === 'path') {
+          (options as Record<string, unknown>)[key] = workingDir;
+        }
```

Comment:
It looks like the logic previously set the default to the workingDir for only properties named `path` but this overrides the value with the workDir for all properties with a `path` format.  I think we might want to keep the `path` name condition to prevent accidentally breaking third-party schematics that could be using the `path` format for other purposes.

We should probably add in a Schematics specific `workingDirectory` concept (on the context maybe?) so we can avoid this whole workaround in the future.

Reply:
https://github.com/angular/angular-cli/pull/23212 is a different approach to this change.